ts-microservices/test/klotho.yaml

```diff
@@ -0,0 +1,61 @@
+app: ts-microservices
+provider: aws
+path: .
+out_dir: compiled
+defaults:
+    execution_unit:
+        type: lambda
+        pulumi_params_by_type:
+            fargate:
+                cpu: 256
+                memory: 512
+            lambda:
+                memorySize: 512
+                timeout: 180
+    static_unit:
+        type: s3
+        pulumi_params_by_type:
+            s3:
+                cloudFrontEnabled: true
+    expose:
+        type: apigateway
+    persist:
+        kv:
+            type: dynamodb
+        fs:
+            type: s3
+        secret:
+            type: s3
+        orm:
+            type: rds_postgres
+            pulumi_params_by_type:
+                rds_postgres:
+                    allocatedStorage: 20
+                    engineVersion: "13.7"
+                    instanceClass: db.t4g.micro
+                    skipFinalSnapshot: true
+        redis:
+            type: elasticache
+            pulumi_params_by_type:
+                elasticache:
+                    nodeType: cache.t3.micro
+                    numCacheNodes: 1
+    pubsub:
+        type: sns
+execution_units:
+    microsrv-api:
+        type: lambda
+        pulumi_params:
+            memorySize: 1024
+            timeout: 120
+    microsrv-users:
+        type: fargate
```

Comment:
are we going to eventually have different configurations/combinations for these config files?

Reply:
yeah thats the hope, but especiialy for ones where we need proxy, etc